sdk_container/src/third_party/coreos-overlay/sys-apps/baselayout/baselayout-9999.ebuild

```diff
@@ -5,18 +5,20 @@ EAPI=7
 EGIT_REPO_URI="https://github.com/flatcar/baselayout.git"
 
 if [[ "${PV}" == 9999 ]]; then
+	inherit git-r3
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
 	EGIT_COMMIT="937a45faef0f7fa88d3d2c3f7ba60a7f3e2e82f7" # flatcar-master
+	SRC_URI="https://github.com/flatcar/baselayout/archive/${EGIT_COMMIT}.tar.gz -> flatcar-${PN}-${EGIT_COMMIT}.tar.gz"
```

Comment:
I remember that we used to have issues with the tarballs generated from a commit that they sometimes changed the hash, because one byte in the tarball has changed or something like that.

Not going to block the PR on the issue - I think we can just react to it when it happens.

Reply:
It has happened on extremely rare occasions, but I know GitHub try very hard to avoid it for precisely this reason. Nearly 9000 Gentoo ebuilds rely on this, so I think we can handle just one.